ui/App/DiagnosticsScreen.svelte

```diff
@@ -0,0 +1,91 @@
+<!--
+ Copyright © 2021 The Radicle Upstream Contributors
+
+ This file is part of radicle-upstream, distributed under the GPLv3
+ with Radicle Linking Exception. For full terms see the included
+ LICENSE file.
+-->
+<script lang="ts">
+  import * as router from "ui/src/router";
+  import { unreachable } from "ui/src/unreachable";
+
+  import BoxIcon from "design-system/icons/Box.svelte";
+  import NetworkIcon from "design-system/icons/Network.svelte";
+  import UserIcon from "design-system/icons/User.svelte";
+
+  import ScreenLayout from "ui/App/ScreenLayout.svelte";
+  import ActionBar from "ui/App/ScreenLayout/ActionBar.svelte";
+  import TabBar from "ui/App/ScreenLayout/TabBar.svelte";
+
+  import ConnectionsTab from "./DiagnosticsScreen/Connections.svelte";
+  import WaitingRoomTab from "./DiagnosticsScreen/WaitingRoom.svelte";
+  import StorageTab from "./DiagnosticsScreen/Storage.svelte";
+
+  export let activeTab: router.DiagnosticsTab;
+
+  const tabs = (active: router.DiagnosticsTab) => [
+    {
+      title: "Storage",
+      active: active === "storage",
+      icon: UserIcon,
+      onClick: () => {
+        router.push({ type: "diagnostics", activeTab: "storage" });
+      },
+    },
+    {
+      title: "P2P Connections",
+      active: active === "connections",
+      icon: NetworkIcon,
+      onClick: () => {
+        router.push({ type: "diagnostics", activeTab: "connections" });
+      },
+    },
+    {
+      title: "Waiting Room",
+      active: active === "waitingRoom",
+      icon: BoxIcon,
+      onClick: () => {
+        router.push({ type: "diagnostics", activeTab: "waitingRoom" });
+      },
+    },
+  ];
+</script>
+
+<style>
+  .container {
+    min-width: var(--content-min-width);
+    margin: 0 auto;
+    padding: 1rem var(--content-padding) 2rem var(--content-padding);
+  }
+
+  .header {
+    min-width: var(--content-min-width);
+    margin: 0 auto;
+    padding: 2rem var(--content-padding);
+    background-color: var(--color-foreground-level-1);
+  }
+</style>
+
+<ScreenLayout contentStyle="padding: 0;">
+  <div class="header">
```

Comment:
I think you need to use the header slot

![2021-11-02_17-15](https://user-images.githubusercontent.com/3919579/139902401-d07926a4-866b-41bf-88e9-c09629dc6856.png)

Reply:
I wanted to avoid the full-size header, but maybe it doesn't matter. Consistency ftw!

2c59285459be1d2b9f8517b06d69383eee9c5d99